python/testing/__init__.py

```
@@ -18,7 +18,7 @@
 
 # TESTING -> CLASS
 class Test(unittest.TestCase):
-    def testCompile(self):
+    def testView(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `testView`

Add return type annotation: `None`

(ANN201)